modules/before-you-begin.adoc

```diff
@@ -0,0 +1,77 @@
+:_module-type: CONCEPT
+
+[id='before-you-begin_{context}']
+= Before you begin
+
+[role='_abstract']
+Before you implement Feature Store in your machine learning workflow, you need the following resources or information:
+
+Knowledge of your data and use case::
+You need knowledge of your use case and your raw underlying data so that you can identify the properties or attributes that you want to define as features. For example, if you are developing ML models that detect possible credit card fraud transactions, you would identify data such as purchase history, transaction location, transaction frequency, or credit limit. 
```

Comment:
```suggestion
You must know your use case and your raw underlying data so that you can identify the properties or attributes that you want to define as features. For example, if you are developing ML models that detect possible credit card fraud transactions, you would identify data such as purchase history, transaction location, transaction frequency, or credit limit. 
```